sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -286,24 +286,24 @@ def to_arrow(self) -> pyarrow.Table:
         return self.client.query(self.query).to_arrow()
 
 
-def block_until_done(client, bq_job):
-    def _is_done(job_id):
-        return client.get_job(job_id).state in ["PENDING", "RUNNING"]
-
+def _block_until_done(client, bq_job):
```

Comment:
Can we keep this function public ? Feast users could potentially use it rather than creating a similar version of it

Reply:
Oh interesting, did not consider that it was intentional to expose it to users. I was just thinking about what would happen if the user did exceed the 30 mins of the retry but the query was still running - I suppose this could be used in that case.